.travis.yml

```
@@ -1,3 +1,6 @@
+---
 language: node_js
-node_js:
-  - "0.10"
+node_js: node
```

```suggestion
node_js: node
dist: xenial
```

Lets update to xenial as well